Documentation/Signoffs/SteeringSystem.md

```diff
@@ -0,0 +1,107 @@
+STEERING WHEEL SUBSYSTEM
+------------------------
+### Function of the Subsystem
+The Steering Wheel subsytem shall recieve the input of the Nissan Leaf steering wheel into a sensor
+embedded into the steering column of the vehicle. The sensor will measure the steering angle and send
+the data into a local microcontroller which will interpret the data and relay it to the simulation.
+
+### Specifications & Constraints
+- Must measure angle of rotation of the steering wheel
+- Must use the Nissan Leaf OEM steering wheel as the input
+- Angle Shall be no more than 960 degrees to the left and right (2 and 2/3 rotations)
+- Error of the angle must be within 1% of the true steering angle
+
+The simulation requires an input for the steering angle of the vehicle. In the Nissan Leaf, the steering
+is limited to 2 and 2/3 rotations from the center point to each clockwise and counter-clockwise direction.
+Therefore, the angle measured must be limited to 960 degrees in each direction. Due to the broader impact 
+accuracy constraint, the error of the angle sensed must be within 1% of the true angle to prevent false
+representation of the steering.
+
+### Schematic
+##### 1. Model
+
+
+![image](https://user-images.githubusercontent.com/100802413/200971670-3beca395-ed8c-400a-b166-59c4ad6b9142.png)
+
+_2D-Model of Gear and Encoder Connection_
+
+![image](https://user-images.githubusercontent.com/100802413/200971713-41b91541-980b-4e55-9a2c-9e852589094b.png)
+
+_3D-Model of Gear and Encoder Connection_
+
+##### 2. Wiring Schematic
+
+
+![image](https://user-images.githubusercontent.com/100802413/200903742-825de5d3-3bc1-40d5-83c7-1e14c02fa940.png)
+
+_Rotary Encoder Pinout_
+
+![image](https://user-images.githubusercontent.com/100802413/200881810-5902fe09-6d5b-472e-89ba-e14607246b68.png)
+
+_Arduino UNO Pinout_
+
+
+### Analysis
```

Comment:
The purpose of this section is to analyze your specific design to show that it will meet the constraints. Remove any additional unnecessary details that aren't directly related to your design, specs, and analysis. Convince me this will do what you say that it needs to by analyzing appropriately.

Reply:
This section is still bloated with unnecessary information. Prove your system will do what you said above and won't violate any constraints. Do it in an organized and explicit fashion.

Reply:
I see very little in the way of analysis here. Prove to me that it will work. That does not require paragraphs of writing. This section should be mostly numbers or simulation results. Include just enough writing so that what you are proving and how you proved it are clear.